pages/data-providers/[data-provider-id]/overview.css

```diff
@@ -0,0 +1,61 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(16rem, auto));
```

Comment:
Can you fix this spacing?

![padding](https://user-images.githubusercontent.com/12775440/72882004-23ea2d80-3d02-11ea-8c56-478fa4dd2eae.PNG)

Reply:
I wish I could. I do not know how to handle this with grid but do not want to go back to flex and use workarounds.

Reply:
I think the only issue here is the wrong number:

```
    grid-template-columns: repeat(auto-fill, minmax(18rem, auto));
```

Reply:
Card has minimal width itself what I would not like to change because it affects flex containers in other places.

There should be another solution but I do not know it yet. I would like to postpone this if you do not mind.